자릿수-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+    const text_n = String(n).split("");
+    const number_n = text_n.map((item) => Number(item));
+    let answer = 0;
+    number_n.forEach((item) => {
+        answer += item;
+    });
+    return answer;
```

Comment:
```suggestion
    const textN = String(n).split("");
    const numberN = textN.map((item) => +item);
    
    return number_n.reduce((acc, cur) => acc + cur, 0);
```
프로그래밍 언어에서 대표적인 변수 선언 방법은 카멜케이스와 스네이크 케이스가 있는데 자바스크립트는 기본적으로 카멜케이스를 채택합니다. 그렇기에 메서드 명도 다 카멜케이스로 작성된 거고요. 그리고 배열의 모든 값을 String 타입에서 Number 타입으로 바꾸려고 map을 쓰신거같은데 Number(item) 과 +item 은 같은 기능을 하니까 편하신거 쓰시면 될거같고 자바스크립트에서 reduce라는 메서드가 있는데 보통 누적값을 계산할 때 많이 쓰기 때문에 forEach도 좋지만 reduce도 써보시면 좋을거같아요.